test/tools/jsontests/BlockChainTests.cpp

```diff
@@ -415,9 +437,11 @@ void testBCTest(json_spirit::mObject const& _o)
 {
 	string testName = TestOutputHelper::get().testName();
 	TestBlock genesisBlock(_o.at("genesisBlockHeader").get_obj(), _o.at("pre").get_obj());
-	TestBlockChain blockchain(genesisBlock);
 
-	TestBlockChain testChain(genesisBlock);
+    TestBlockChain::MiningType miningType = getMiningType(_o);
+    eth::IncludeSeal includeSeal = (miningType == TestBlockChain::MiningType::ForceEthash || miningType == TestBlockChain::MiningType::Default) ? WithSeal : WithoutSeal;
```

Comment:
Why now we include seal for `Default`, too? What if default changes, will you remember to change this line? (I would not)

Reply:
Then tests will fail. By default blockchain tests are done with PoW. 
I told you that we should better use explicit constructor with mining type + in unittests.
So that you could remove  HomsteadTestNoProof chain configs.